packages/core/src/Asset/util.ts

```diff
@@ -1,33 +1,24 @@
 import { CardanoSerializationLib } from '@cardano-sdk/cardano-serialization-lib';
 import { AssetName, ScriptHash } from '@emurgo/cardano-serialization-lib-nodejs';
+import { Buffer } from 'buffer';
 
-export const policyIdFromAssetId = (assetId: string, bech32PrefixLength = 4): string =>
-  assetId.slice(0, 52 + bech32PrefixLength);
-export const assetNameFromAssetId = (assetId: string, bech32PrefixLength = 4): string =>
-  assetId.slice(52 + bech32PrefixLength);
+export const policyIdFromAssetId = (assetId: string): string => assetId.slice(0, 56);
+export const assetNameFromAssetId = (assetId: string): string => assetId.slice(56);
 
-export const createAssetSerializer = (csl: CardanoSerializationLib, bech32Prefix = 'b32_') => {
-  const encoder = new TextEncoder();
-  const decoder = new TextDecoder();
+const textDecoder = new TextDecoder();
+const textEncoder = new TextEncoder();
+
+/**
+ * @returns {string} concatenated policy id and asset name
+ */
+export const createAssetId = (scriptHash: ScriptHash, assetName: AssetName): string =>
+  Buffer.from(scriptHash.to_bytes()).toString('hex') + textDecoder.decode(assetName.name());
```

Comment:
`assetName` shouldn't be decoded. The only time we would decode is for presentation purposes, but there's no enforcement in the ledger so it's not guaranteed to be UTF-8.

Reply:
I see. We still need to stringify it. How about `[...assetName].join('')`?

Reply:
```
Buffer.from(asset.name()).toString("hex");
```